mangaki/mangaki/static/js/vote.js

```diff
@@ -138,11 +138,14 @@ Card.prototype.hydrate = function (work) {
 
   // FIXME: We should use a solution that doesn't require monkey-patching HTML
   // with jQuery in the future, but for now that is the easiest way.
+  var card_url = '/' + this.category + '/' + work.id;
+  console.log(work);
   $el.find('.work-card__media').css('background-image', 'url("' + work.poster + '")');
-  $el.find('.work-card__title').text(work.title);
-  $el.find('.work-cover__wrapper').attr('data-category', work.category);
+  $el.find('.work-card__title').html('<a href="' + card_url + '">' + work.title + '</a>');
```

Comment:
Don't you want to select the inner `<a />` element and change its attribute `href`, then, change its text, something along `$el.find('.work-card__title').find('a')` would work surely, using a better selector, you could get it in one shot. My selector skills are a bit rusty, something like: `a < .work-card__title` could work, I don't remember.